app/Controllers/PluginController.php

```
@@ -393,6 +361,37 @@ public function updateSettings(Request $request, Response $response, array $args
                     'enabled' => $enabled
                 ]
             ]));
+        } elseif ($plugin['name'] === 'z39-server') {
+            // Z39.50/SRU Integration settings
+            $enableServer = isset($settings['enable_server']) && $settings['enable_server'] === '1';
+            $enableClient = isset($settings['enable_client']) && $settings['enable_client'] === '1';
+            $servers = $settings['servers'] ?? '[]';
+
+            // Validate JSON
+            if (is_string($servers)) {
+                $decoded = json_decode($servers, true);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$decoded'. (undefined)

(UnusedLocalVariable)